### ISSUE TITLE
estimating abundance in numbers

### ISSUE THREAD

**Issue by EllenY** [nwfsc-assess/geostatistical_delta-GLMM#8]
Hi,
How would I modify the R code to estimate abundance in numbers and plot surface densities in numbers instead of weight?
Thank you,

**Comment by James-Thorson** [comment]
Analyzing count valued data is already an option, and involves specifying the using the ObsModel input to Data_Fn.  You can find the list of available distributions, including count-valued distributions, using:

```R
?SpatialDeltaGLMM::Data_Fn
````

after re-installing the package to use the most recent ('development') version.

**Comment by EllenY** [comment]
Thank you. Do I used this script to re-install the most recent package?
devtools::install_github("nwfsc-assess/geostatistical_delta-GLMM")

**Comment by James-Thorson** [comment]
Yes, that's the script, and you can also install past releases by
specifying the optional "ref" input

jim
